lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -3030,17 +3030,22 @@ def initialize(self, connection):
         # https://www.postgresql.org/docs/9.3/static/release-9-2.html#AEN116689
         self.supports_smallserial = self.server_version_info >= (9, 2)
 
-        std_string = connection.exec_driver_sql(
-            "show standard_conforming_strings"
-        ).scalar()
-        self._backslash_escapes = std_string == "off"
+        self.set_backslash_escapes()
```

Comment:
I think you need to pass the connection here

Reply:
Derp. Thanks for catching that. My IDE wasn't done indexing sqlalchemy yet so the linter wasn't working yet xD